src/plugins/dashboard/public/dashboard_actions/_dashboard_actions_strings.ts

```diff
@@ -117,13 +117,17 @@ export const dashboardReplacePanelActionStrings = {
     i18n.translate('dashboard.panel.removePanel.replacePanel', {
       defaultMessage: 'Replace panel',
     }),
-  getSuccessMessage: (savedObjectName: string) =>
-    i18n.translate('dashboard.addPanel.savedObjectAddedToContainerSuccessMessageTitle', {
-      defaultMessage: '{savedObjectName} was added',
-      values: {
-        savedObjectName,
-      },
-    }),
+  getSuccessMessage: (savedObjectName?: string) =>
+    savedObjectName
+      ? i18n.translate('dashboard.addPanel.savedObjectAddedToContainerSuccessMessageTitle', {
+          defaultMessage: '{savedObjectName} was added',
+          values: {
+            savedObjectName: `'${savedObjectName}'`,
+          },
+        })
+      : i18n.translate('dashboard.addPanel.panelAddedToContainerSuccessMessageTitle', {
+          defaultMessage: 'A panel was added',
```

Comment:
Do we need a toast when the panel is added? Adding other panels like this doesn't show a toast, this is only used when you add Library visualizations.

Reply:
I moved the code that adds an embeddable and calls this toast from `src/plugins/dashboard/public/dashboard_app/top_nav/editor_menu.tsx` to  `src/plugins/dashboard/public/dashboard_app/top_nav/dashboard_editing_toolbar.tsx`  to use it for creating image embeddable. The toast came with the code. 

Do you think I should add an if/else there to not show a toast in case this wasn't a library viz? I personally not sure why the behavior has to be different.

Reply:
Ah, makes sense! I didn't realize that you standardized these calls. Thanks Anton!